gui/jails/forms.py

```diff
@@ -475,19 +475,20 @@ def save(self):
         self.instance = Jails.objects.get(jail_host=jail_host)
 
 
-def is_JailRoot_Shared(jail_root):
+def is_jail_root_shared(jail_root):
     paths = [c.cifs_path for c in CIFS_Share.objects.all()]
     paths.extend([a.afp_path for a in AFP_Share.objects.all()])
     paths.extend([n.path for n in NFS_Share_Path.objects.all()])
     paths.extend([w.webdav_path for w in WebDAV_Share.objects.all()])
-    iscsi_target_paths = [i.iscsi_target_extent_path for i in iSCSITargetExtent.objects.all()]
-    for i in range(len(iscsi_target_paths)):
-        iscsi_target_paths[i] = iscsi_target_paths[i].rpartition('/')[0]
-    paths.extend(iscsi_target_paths)
+    extents = iSCSITargetExtent.objects.all()
+    for e in extents:
+        if e.iscsi_target_extent_type == 'File':
```

Comment:
You should use ORM filter instead, e.g.:
extents = iSCSITargetExtent.objects.filter(iscsi_target_extent_type='File')